modules/nw-multinetwork-sriov.adoc

```diff
@@ -40,247 +62,386 @@ and device ID 0x1015
+the interface list, you can find the information of the network devices.  
+
+You should never have to modify this CRD, nor the CRs. 
+
+The following is an example of a typical Custom Resource for
+`SriovNetworkNodeState`.
+
+[source,yaml]
 ----
-kind: DaemonSet
-apiVersion: apps/v1
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodeState
 metadata:
-  name: sriov-cni
-  namespace: openshift-sriov
-  annotations:
-    kubernetes.io/description: |
-      This daemon set launches the SR-IOV CNI plugin on SR-IOV capable worker nodes.
+  creationTimestamp: "2019-08-27T06:01:36Z"
+  generation: 1
+  name: node-25
+  namespace: sriov-network-operator
+  ownerReferences:
+  - apiVersion: sriovnetwork.openshift.io/v1
+    blockOwnerDeletion: true
+    controller: true
+    kind: SriovNetworkNodePolicy
+    name: default
+    uid: 20c3c0f9-c890-11e9-91f7-0028ccd628ee
+  resourceVersion: "2639363"
+  uid: 2103a9fe-c890-11e9-91f7-0028ccd628ee
 spec:
-  selector:
-    matchLabels:
-      app: sriov-cni
-  updateStrategy:
-    type: RollingUpdate
-  template:
-    metadata:
-      labels:
-        app: sriov-cni
-        component: network
-        type: infra
-        openshift.io/component: network
-    spec:
-      nodeSelector:
-        kubernetes.io/os: linux
-      tolerations:
-      - operator: Exists
-      serviceAccountName: sriov-cni
-      containers:
-      - name: sriov-cni
-        image: quay.io/openshift/{image-prefix}-sriov-cni:v4.0.0
-        securityContext:
-          privileged: true
-        volumeMounts:
-        - name: cnibin
-          mountPath: /host/opt/cni/bin
-      volumes:
-        - name: cnibin
-          hostPath:
-            path: /var/lib/cni/bin
-----
-
-. Run the following command to create the `sriov-cni` DaemonSet:
-+
-----
-$ oc create -f sriov-cni.yaml
+  dpConfigVersion: d41d8cd98f00b204e9800998ecf8427e
+status:
+  interfaces:
+  - deviceID: "1017"
+    driver: mlx5_core
+    mtu: 1500
+    name: ens785f0
+    pciAddress: "0000:18:00.0"
+    totalvfs: 8
+    vendor: 15b3
+  - deviceID: "1017"
+    driver: mlx5_core
+    mtu: 1500
+    name: ens785f1
+    pciAddress: "0000:18:00.1"
+    totalvfs: 8
+    vendor: 15b3
+  - deviceID: 158b
+    driver: i40e
+    mtu: 1500
+    name: ens817f0
+    pciAddress: 0000:81:00.0
+    totalvfs: 64
+    vendor: "8086"
+  - deviceID: 158b
+    driver: i40e
+    mtu: 1500
+    name: ens817f1
+    pciAddress: 0000:81:00.1
+    totalvfs: 64
+    vendor: "8086"
+  - deviceID: 158b
+    driver: i40e
+    mtu: 1500
+    name: ens803f0
+    pciAddress: 0000:86:00.0
+    totalvfs: 64
+    vendor: "8086"
+  syncStatus: Succeeded
 ----
 
-== Configuring additional interfaces using SR-IOV
+== Configuring SR-IOV network devices
+
+The SR-IOV Network Operator introduces `SriovNetworkNodePolicy` Custom Resource
+Definition (CRD) to define the SR-IOV network device and the configuration of
+SR-IOV device plug-in
```

Comment:
add full stop at the end of this paragraph.

Reply:
Updated.